### PR TITLE
[data-cat][xs]: added tests for data cat tsv files - refs datahq/data…

### PR DESCRIPTION
**Pull request by Mikanebu**
- tests for data cat `tsv` remote/local files

### CODE REVIEW

File: test/cli.test.js [hunk 1]
```diff
@@ -501,6 +501,20 @@ test('cat command - remote csv file', async t => {
   t.true(stdout[3].includes('│ 1.0  │'))
 })
 
+test('cat command - local tsv file', async t => {
+  const url_ = 'test/fixtures/test-data/files/csv/separators/tab.tsv'
```
Comment (anuveyatsu) [inline test/cli.test.js]: it's `path_` here

Comment (Mikanebu) [reply test/cli.test.js]: Good catch! It will be fixed.

**Review by anuveyatsu — CHANGES_REQUESTED** [review]
Can you please move these tests down to somewhere after QA tests for different encodings? Otherwise, you've placed them inside QA tests for basic csv

**Comment by Mikanebu** [comment]
@anuveyatsu moved

**Review by anuveyatsu — APPROVED** [review]
Great!

**Pull request merged**